src/api/faq/faq.controller.js

```diff
@@ -43,4 +43,42 @@ async function getFAQ(req, res) {
   }
 }
 
-module.exports = { getFAQ };
+async function addFAQ(req, res) {
+  try {
+    const { question, answer } = req.body;
+    if (!question || !answer) throw new ReferenceError('Make sure your request includes a question and answer.');
+    await new FAQ({ question, answer }).save();
+    return res.json({ message: 'FAQ added successfully to DB.' });
```

Comment:
Should this include a status code in the response?
`{ message: 'FAQ added successfully to DB.', status: 200 }`